Misc/NEWS.d/next/Library/2024-09-24-22-38-51.gh-issue-123884.iEPTK4.rst

```diff
@@ -0,0 +1,4 @@
+Fixed bug in itertools.tee() handling of other tee inputs (a tee in a tee).
```

Comment:
Maybe you should use **:func:** to modify `itertools.tee()`, like this
```
:func:`itertools.tee()`
```

Reply:
This is a backport. If you want to propose changes not specific to `3.13`, please create a PR against `main`.